packages/framework/src/Component/EntityLog/EventListener/EntityLogEventListener.php

```diff
@@ -99,15 +96,11 @@ protected function log(string $action, object $entity): void
     {
         $loggableSetup = $this->loggableEntityConfigFactory->getLoggableSetupByEntity($entity);
 
-        try {
-            if (!$loggableSetup->isLoggable()) {
-                return;
-            }
-
-            $this->registerLog($entity, $loggableSetup, $action);
-        } catch (Throwable $exception) {
```

Comment:
I would rather keep this catch here. I do not want the application to crash when there is an error in entity logging - I prefer the way it was - it will silently add an entry into the log.